build/openwisp_freeradius/raddb/mods-available/sqlcounter

```diff
@@ -0,0 +1,41 @@
+# The dailycounter is included by default in the freeradius conf
+
+sqlcounter dailycounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
+
+    counter_name = Daily-Session-Time
+    check_name = Max-Daily-Session
+    reply_name = Session-Timeout
+
+    key = User-Name
+    reset = daily
+
+    $INCLUDE ${modconfdir}/sql/counter/${dialect}/${.:instance}.conf
+}
+
+# The noresetcounter is included by default in the freeradius conf
+sqlcounter noresetcounter {
+    sql_module_instance = sql
+    dialect = ${modules.sql.dialect}
+
+    counter_name = Max-All-Session-Time
+    check_name = Max-All-Session
+    key = User-Name
+    reset = never
+
+    $INCLUDE ${modconfdir}/sql/counter/${dialect}/${.:instance}.conf
+}
+
+# The dailybandwidthcounter is added for django-freeradius
+sqlcounter dailybandwidthcounter {
+   counter_name = Max-Daily-Session-Traffic
+   check_name = Max-Daily-Session-Traffic
+   sql_module_instance = sql
+   key = 'User-Name'
+   reset = daily
+   query = "SELECT SUM(acctinputoctets + acctoutputoctets) \
+            FROM radacct \
+            WHERE UserName='%{${key}}' \
+            AND UNIX_TIMESTAMP(acctstarttime) + acctsessiontime > '%%b'"
+}
```

Comment:
what happens if we want to modify any of these freeradius configuration files? Can we do it without altering the source code of the image?

Reply:
I think you would have to mount over the raddb directory or mount individual files as needed.

Reply:
I agree, the default rabbd/ files are just sample of basic configuration for testing (and can be used as an example for user), user can mount their own raddb directory for customization. :smile:

Reply:
Added documentation for the same. :smile:

Reply:
Check the docs for Freeradius/Kubernetes, it looks like it has some copy paste from the Nginx section.

Reply:
Oh, thanks! Fixed it! :smile: